etna/docker/app/docker-entrypoint.sh

```diff
@@ -4,6 +4,7 @@ set -e
 : "${RANDOM_MAX:=500}"
 [ -n "$DEBUG" ] && echo "Running: $@"
 export PATH="/app/node_modules/.bin:/app/vendor/bundle/$RUBY_VERSION/bin:$PATH"
+cp npmrc ~/.npmrc || true
```

Comment:
I am working soon on a way to 'share' these docker pieces between projects.  Right now I have to manually duplicate new logic to all projects.